specs/altair/beacon-chain.md

```diff
@@ -335,6 +336,37 @@ def get_unslashed_participating_indices(state: BeaconState, flag_index: int, epo
     return set(filter(lambda index: not state.validators[index].slashed, participating_indices))
 ```
 
+#### `get_attestation_participation_flags`
+
+```python
+def get_attestation_participation_flag_indices(state: BeaconState,
```

Comment:
I like this refactoring! IMHO we can apply it even if the `translate_participation` solution doesn't get included.

Reply:
Thanks for the review!